administrator/components/com_menus/tmpl/item/edit_modules.php

```diff
@@ -98,37 +98,20 @@ class="btn btn-link module-edit-link"
 					data-module-id="<?php echo $module->id; ?>">
 					<?php echo $this->escape($module->title); ?></button>
 			</td>
+			<td id="menus-<?php echo $module->id; ?>" style="width:15%" >
```

Comment:
not the right one on this
but
should'nt the css be putted some where else

Reply:
@alikon all other columns use the css in the same way 
I have made no changes to the css

Reply:
that's why i'm asking you

> all other columns use the css in the same way

this is not the correct answer :smile: 
what should be the correct way to do this ?

Reply:
> what should be the correct way to do this ?
Sure will look into this.

@alikon is the show/hide switch correct?

Reply:
tbh i didn't test yet ....but looks cool....if  done in the right way